model-client/src/commonMain/kotlin/org/modelix/model/client2/IModelClientV2.kt

```diff
@@ -53,5 +53,7 @@ interface IModelClientV2 {
 
     suspend fun pull(branch: BranchReference, lastKnownVersion: IVersion?): IVersion
 
+    suspend fun pullHash(branch: BranchReference): String
+
     suspend fun poll(branch: BranchReference, lastKnownVersion: IVersion?): IVersion
```

Comment:
How do pull and poll differ here and why is there only a `pullHash` but not a `pollHash` function?